### DISCUSSION TITLE
EXT3 - Remove CW - week view & abbr. day of week to single letters

### DISCUSSION THREAD

**Discussion by lefebe** [MMRIZE/MMM-CalendarExt3#12] [Q&A] [answered]
Hey all!  I am diving into MM and this may be a pretty obvious question.  

I've got most of the css and config where I want it, but I can't seem to get rid of the current week count.  Can anyone point me in the right direction?

I would also like to change the days of the week across the top to single letters (i.e. M vs Mon)

TIA!

**Comment by eouia** [comment, marked as answer]
1. Add this into your `custom.css`
```css
.CX3 .cw {
  display: none;
}
```

2. The default `headerWeekDayOptions` would be `{weekday: long}`. You can change it to `{weekday: 'short'}` or `{weekday: 'narrow'}`. But the representation is also affected by your `locale`. So in some culture (locale), `short` or `narrow` format is not perfectly matched to your ideal expectation.
